pkg/gofr/datasource/opentsdb/version.go

```diff
@@ -0,0 +1,62 @@
+package opentsdb
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"go.opentelemetry.io/otel/trace"
+)
+
+// VersionResponse is the struct implementation for /api/version.
+type VersionResponse struct {
+	StatusCode  int
+	VersionInfo map[string]string `json:"VersionInfo"`
+	logger      Logger
+	tracer      trace.Tracer
+	ctx         context.Context
+}
+
+func (verResp *VersionResponse) SetStatus(code int) {
+	setStatus(verResp.ctx, verResp, code, "SetStatus-VersionResp", verResp.logger)
+}
+
+func (verResp *VersionResponse) setStatusCode(code int) {
+	verResp.StatusCode = code
+}
+
+func (verResp *VersionResponse) String() string {
+	return toString(verResp.ctx, verResp, "ToString-VersionResp", verResp.logger)
+}
+
+func (verResp *VersionResponse) GetCustomParser() func(respCnt []byte) error {
+	return getCustomParser(verResp.ctx, verResp, "GetCustomParser-VersionResp", verResp.logger,
+		func(resp []byte) error {
+			return json.Unmarshal([]byte(fmt.Sprintf(`{"VersionInfo":%s}`, string(resp))), &verResp)
+		})
+}
```

Comment:
I who have expected the same logic to use a map[string]any, use unmarshal then set VersionInfo instead of this

Maybe there are still other remaining in the code. Could you check?